### DISCUSSION TITLE
AI suggest Javascript snippets for C#

### DISCUSSION THREAD

**Discussion by PerfectlyFineCode** [community/community#19382] [Copilot Conversations] [answered]
Code

```cs
// Write Hello World to console output



Console.ReadLine();
```
![image](https://user-images.githubusercontent.com/70201838/175649471-a5d0ddaf-4670-48a0-b17f-ef5682358cc4.png)

**Comment by melvincwng** [comment, marked as answer]
@PerfectlyFineCode, for bugs, kindly report it to [Github Bug Report](https://support.github.com/contact/bug-report) over here.

Hope this clarifies.

**Comment by hsenag** [comment]
Hi, can you confirm you're using Visual Studio here?
